cluster/gce/gci/configure-helper.sh

```diff
@@ -880,6 +880,18 @@ function compute-master-manifest-variables {
   fi
 }
 
+# A helper function that bind mounts kubelet dirs for running mount in a chroot
+function prepare-mounter-rootfs {
+  echo "Prepare containerized mounter"
+  mount --bind "${CONTAINERIZED_MOUNTER_HOME}" "${CONTAINERIZED_MOUNTER_HOME}"
```

Comment:
It is a copy-paste thing, I assumed it is for remount. But you are right, it is not necessary, I think